k8s/spack/builds-spack-io/ingress.yaml

```
@@ -0,0 +1,24 @@
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: builds-spack-io
+  namespace: spack
```

Here are the changes that I had to apply to your ingress in order to get around the `413` size limit issue.

```suggestion
metadata:
  name: builds-spack-io
  namespace: spack
  annotations:
    nginx.ingress.kubernetes.io/proxy-body-size: "10024M"
    nginx.ingress.kubernetes.io/client-body-buffer-size: "10024M"
    nginx.ingress.kubernetes.io/proxy-connect-timeout: "120"
    nginx.ingress.kubernetes.io/proxy-send-timeout: "120"
    nginx.ingress.kubernetes.io/proxy-read-timeout: "120"
```

---

I love that I can control this directly from my configs! Let's give it a try. I'm building new containers with the migrations and will do another deploy after that.